documentation/MIGRATION-guide-for-next-generation-management-libraries.md

```diff
@@ -6,7 +6,7 @@ This document is intended to show the customers of the JavaScript/TypeScript man
 
 ## Current status
```

Comment:
For the file name, can we make it simpler like MigrationGuide_ManagementSDK

Reply:
Not sure if that's a normal typescript filename pattern, @nickzhums what do you think of it ?

Reply:
what's the normal typescript pattern?

Reply:
For code, it's camel case I think, not sure about the document name.